.vscode/settings.json

```diff
@@ -0,0 +1,12 @@
+{
+    "editor.formatOnSave": true,
+    "editor.renderWhitespace": "boundary",
+    "editor.rulers": [
+        80
+    ],
+    "eslint.autoFixOnSave": true,
+    "eslint.enable": true,
+    "files.trimTrailingWhitespace": true,
+    "flow.useNPMPackagedFlow": true,
```

Comment:
This is a great example of why we should have `extensions.json`, as that way we can recommend that the `flowtype` extension is installed.